content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,182 @@
+Next, set the :guilabel:`Date range`, :guilabel:`Company`, :guilabel:`Country`, :guilabel:`Region`,
+:guilabel:`Options`, :guilabel:`Location code`, :guilabel:`Tax code`, or :guilabel:`Tax type` under
+the :guilabel:`Select report details` section.
+
+Depending on the report size, the following two options are available in the section labeled,
+:guilabel:`Select the approximate number of transactions for your report`: :guilabel:`Create and
+download the report instantly` (for small reports) and :guilabel:`Create and download the report in
+the background` (for larger reports). Select one or the other depending on the volume of
+transactions in this report.
+
+Finally, make a selection of the file type to download. Either a :guilabel:`.PDF` or
+:guilabel:`.XLS` can be chosen.
+
+After making all the configurations, click :guilabel:`Create report` to download the report. Click
+:icon:`fa-star-o` :guilabel:`Make this report a favorite` to save the report configuration to the
+user's favorites.
+
+.. tip::
+   Optionally, select a pre-configured report from the :guilabel:`Frequently used reports` section
+   of the reporting dashboard. Access reporting by clicking on the :guilabel:`Reports` option in the
+   top menu of the Avalara management console.
+
+Add more jurisdictions
+======================
+
+Additional jurisdictions can be added (tax locations) in the Avalara management console. Navigate to
+either Avalara's `sandbox <https://sandbox.admin.avalara.com/>`_ or `production
+<https://admin.avalara.com/>`_ environment. This will depend on which type of account was set in the
+:doc:`integration <../avatax>`.
+
+Next, navigate to :menuselection:`Settings --> Where you collect taxes`. Choose between the three
+different tabs depending on the business need. The first tab is :guilabel:`Sales and use tax` where
+tax can be collected for the United States. Click the :icon:`fa-plus` :guilabel:`Add to where you
+collect sales and use tax` to add another location where the company collects sales and use tax.
+
+The second option, is the :guilabel:`VAT / GST` tab where the :icon:`fa-plus` :guilabel:`Add a
+country or territory where you collect VAT / GST` can be selected to add another country or
+territory where the company collects VAT/GST.
+
+Finally, on the far-right is the :guilabel:`Customs Duty` tab, where a country can be added where
+the company collects customs duty. Simply click on the :icon:`fa-plus` :guilabel:`Add a country
+where you calculate customs duty` icon below the tab.
+
+.. image:: avalara_portal/where-you-collect-tax.png
+   :align: center
+   :alt: AvaTax management console, on the Where you collect tax page, with the add button and
+         sales and use tax tab highlighted.
+
+Tax exemption certificate
+=========================
+
+Tax exemption certificates for customers can be added into the Avalara management console so that
+*AvaTax* is aware of which customers may be exempt from paying certain taxes. To add an *exception
```

Comment:
Just a general idea- what is the difference between Avalara and Avatax? It could be aded in the intro, something like "X is the parent cmpany, and Y is the program..." etc. I have NO idea which is what, but osmething liek that could clear up these words, because the names seem interchangebale/related but I am not sure how or why. Just a little extra nugget- could be a note or something at the top?